source/agora/consensus/Ledger.d

```diff
@@ -556,43 +556,34 @@ public class Ledger
 
     /***************************************************************************
 
-        Create the Coinbase TX for this nomination round and append it to the
-        tx_set
+        Create the Coinbase TX for this payout block and append it to the tx_set
 
         Params:
             height = block height
             tot_fee = Total fee amount (incl. data)
             tot_data_fee = Total data fee amount
-            missing_validators = MPVs
 
         Returns:
             List of expected Coinbase TXs
 
     ***************************************************************************/
 
-    public Transaction[] getCoinbaseTX (in Height height, in Amount tot_fee, in Amount tot_data_fee,
-        in uint[] missing_validators) nothrow @safe
+    public Transaction[] getCoinbaseTX (in Height height, in Amount tot_fee, in Amount tot_data_fee)
+        nothrow @safe
     {
-        UTXO[] stakes;
-        this.enroll_man.getValidatorStakes(height, &this.utxo_set.peekUTXO, stakes,
-            missing_validators);
-        const commons_fee = this.fee_man.getCommonsBudgetFee(tot_fee,
-            tot_data_fee, stakes);
+        if (height % this.params.PayoutPeriod != 0)    // This is not Coinbase payout block
+            return null;
 
         Output[] coinbase_tx_outputs;
-        // pay the commons budget
-        if (commons_fee > Amount(0))
-            coinbase_tx_outputs ~= Output(commons_fee,
-                this.params.CommonsBudgetAddress, OutputType.Coinbase);
 
-        // pay the validator for the past blocks
+        // pay the validator and Commons Budget for the past blocks
         if (auto payouts = this.fee_man.getAccumulatedFees(height))
             foreach (pair; payouts.byKeyValue())
-                if (pair.value > Amount(0))
+                // TODO: uncomment when fees are calculated in TxBuilder
+                // if (pair.value > Amount(0))
```

Comment:
Temporary until fees are calculated in TxBuilder

Reply:
Is removed in https://github.com/bosagora/agora/pull/2370 which has to be merged after this one.